File: main_py.py
import json

GET_RAW_PATH = "/birthDate"

def lambda_handler(event, context):
    print(event)
    if event['rawPath'] == GET_RAW_PATH:
        print('Received birthDate request')
        personId = event['queryStringParameters']['date']
        print("with param date =" + date)
        return { "Birth date:": + date, }

File: main_py.js
exports.handler = async (event) => {
    const date = event.queryStringParameters?.date || "NA";
    const body = { success: true, message: 'Birth date is: ' + date };
    const response = {
        statusCode: 200,
        body: JSON.stringify(body),
    };
    return response;
};
